examples/middleware-matcher/middleware.js

```diff
@@ -11,5 +11,8 @@ export default function middleware(req) {
 }
 
 export const config = {
-  matcher: ['/public/disclaimer', '/((?!public|static).*)'],
+  matcher: [
+    '/public/disclaimer', // match a single, specific page
```

Comment:
This might be confusing calling it a page but it starts with `public` which may cause users to think matching files in the `public` folder must be prefixed here which isn't the case. 
```suggestion
    '/disclaimer', // match a single, specific page
```